src/rwe/GameScene.cpp

```diff
@@ -628,26 +625,27 @@ namespace rwe
                                     localPlayerIssueUnitOrder(selectedUnit, AttackOrder(*coord));
                                     cursorMode.next(NormalCursorMode());
                                 }
-                            }
-                        }
+                            },
+                            [](auto) {});
                     }
                 },
                 [&](const MoveCursorMode&) {
                     for (const auto& selectedUnit : selectedUnits)
                     {
-                        auto coord = getMouseTerrainCoordinate();
-                        if (coord)
-                        {
-                            if (isShiftDown())
-                            {
-                                localPlayerEnqueueUnitOrder(selectedUnit, MoveOrder(*coord));
-                            }
-                            else
-                            {
-                                localPlayerIssueUnitOrder(selectedUnit, MoveOrder(*coord));
-                                cursorMode.next(NormalCursorMode());
-                            }
-                        }
+                        match(
+                            elementOfInterest,
```

Comment:
This refactoring makes it impossible to issue a move order to a position that is already occupied by a unit when in move cursor mode. I think this `elementOfInterest` work needs to be reverted (and shouldn't be necessary anyway to implement repairing)

In fact, during testing I observed that it is impossible to issue an order in move mode at all anymore. Something has gone wrong here.